svea_vision/scripts/object_detect.py

```diff
@@ -220,17 +207,6 @@ def callback(self, image):
             object_array.objects = objects
             self.pub_objects.publish(object_array)
 
-        if self.ENABLE_BBOX_IMAGE:
```

Comment:
Should this be removed? If yes, did you remove relevant parts at initialization?